centreon/packages/js-config/cypress/e2e/commands.ts

```diff
@@ -399,6 +399,38 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add('insertDashboardWithPatch', (dashboardBody, patchBody) => {
```

Comment:
the function itself is correct and good, but you probably should find a more appropriate name for the function (like insertDashboardWithWidget)